### ISSUE TITLE
Bug in levels

### ISSUE THREAD

**Issue by wispoz** [dabeng/OrgChart#314]
In hybrid mode if 
verticalDepth > depth levels not expands.
How reproduce: 
in https://rawgit.com/dabeng/OrgChart/master/demo/vertical-depth.html
set 
```js
    $('#chart-container').orgchart({
      'data' : datascource,
      'nodeContent': 'title',
      'verticalDepth': 4, //<-this
      'depth':3 //<-this
    });
```
and try expand 'Hei Hei' and you get issue 
![screenshot 19](https://user-images.githubusercontent.com/156639/33477997-d8337c42-d698-11e7-9f60-4441767f9c70.png)

**Comment by wispoz** [comment]
any ideas ?)

**Comment by dabeng** [comment]
Hi @wispoz , this bug has been fixed in new version 2.0.11. You can play around [hybrid chart demo](https://codepen.io/dabeng/pen/XVJemd) on codepen.io.
